src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.Netcore.csproj

```diff
@@ -24,7 +24,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net472'">
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.18.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.21.0-preview" />
```

Comment:
You will need to re-publish this client for the new runtime and newtonsoft.json version